packages/daemon/package.json

```diff
@@ -52,7 +52,7 @@
     "clear": "midway-bin clean",
     "dev": "cross-env NODE_ENV=local midway-bin dev --port=6927 --ts",
     "debug": "cross-env NODE_ENV=local midway-bin debug --port=6927 --ts",
-    "cov": "midway-bin cov --ts",
+    "cov": "midway-bin cov --ts -x 'src/migrations/'",
```

Comment:
What does this fix up?

Reply:
Ignore the migration script for coverage, we should remove this when we find the way to test them.

Reply:
This is to exclude `migrations` in the coverage as we do not implement `unit test` for them yet.